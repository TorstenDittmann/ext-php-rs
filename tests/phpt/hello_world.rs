extern crate ext_php_rs;
use ext_php_rs::prelude::*;

#[php_function]
pub fn hello_world() -> String {
    "Hello world".into()
}

#[php_module]
pub fn module(module: ModuleBuilder) -> ModuleBuilder {
    module
}
