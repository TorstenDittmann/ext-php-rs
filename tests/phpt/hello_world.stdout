string(11) "Hello world"
